pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SparkRBackendHandler.scala

```diff
@@ -144,6 +144,7 @@ class SparkRBackendHandler(server: SparkRBackend) extends SimpleChannelInboundHa
       args: Array[java.lang.Object],
       parameterTypes: Array[Class[_]]): Boolean = {
     if (parameterTypes.length != numArgs) {
+      // println(s"num of arguments numArgs not match with ${parameterTypes.length}")
```

Comment:
It's annoying  to see many logging about this, because it's normal to have multiple signature for a API with different number arguments. With `println`, we could not mute it, so I'd like to disable is by default.